src/org/opendatakit/briefcase/reused/TriStateBoolean.java

```diff
@@ -17,10 +17,6 @@
     this.booleanValue = booleanValue;
   }
 
-  public static TriStateBoolean of(boolean value) {
```

Comment:
new/revisit

I question myself why aren't we using this factory? Maybe we're doing too much work elsewhere. Maybe we need to lose the factory :man_shrugging: